### DISCUSSION TITLE
在mpvue中使用lin-ui为什么所有组件的事件属性都不能使用?

### DISCUSSION THREAD

**Discussion by Mark-AQ** [TaleLin/lin-ui#1334] [求助] [answered]
![image](https://user-images.githubusercontent.com/13758334/109934181-31e0ef00-7d07-11eb-9595-3244e1df21ef.png)

![image](https://user-images.githubusercontent.com/13758334/109934229-3e654780-7d07-11eb-9b8b-77a37a50e327.png)

报错Component "pages/home/main" does not have a method "changeTabs" to handle event "linchange".

**Comment by juzi214032** [comment, marked as answer]
这应该是 mpvue 的问题
